data/yam/agama/auto/sle_lvm.jsonnet

```diff
@@ -0,0 +1,36 @@
+{
+  "product": {
```

Comment:
jsonnet is very similar to json if we don't do anything fancy, but please remove double quotes `"` all over the place (we will fix this in other files in another ticket).
See jsonnet docs: https://jsonnet.org/

Reply:
Done, thanks for review.